db/migrate/20171006133132_rename_assignment_journal_to_feedback.rb

```diff
@@ -0,0 +1,9 @@
+class RenameAssignmentJournalToFeedback < ActiveRecord::Migration[5.1]
+  def up
+    rename_table :assignment_journals, :feedbacks
+  end
+
+  def down
+    rename_table :feedbacks, :assignment_journals
+  end
```

Comment:
You don't really need the separate `up` / `down` methods, you can just define `change` and Rails knows how to reverse `rename_table` (see https://github.com/rails/rails/blob/master/activerecord/lib/active_record/migration/command_recorder.rb for a list of supported methods)

Reply:
Ah, I guess I did that out of the intend to be a good boy, after RuboCop told me to do up and down in `20171006140658_modify_feedbacks_to_be_polymorphic.rb` 😉 
I'll change that.